src/Keboola/StorageApi/Client/LogMessageFormatter.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\StorageApi\Client;
+
+use GuzzleHttp\MessageFormatterInterface;
+use Psr\Http\Message\RequestInterface;
+use Psr\Http\Message\ResponseInterface;
+use Throwable;
+
+class LogMessageFormatter implements MessageFormatterInterface
```

Comment:
RequestLogMessageFormatter ?

Reply:
je mi to asi fuk.. reknete si

Reply:
`RequestLogMessageFormatter ` dava smysl